src/trans/gpu/algor/hicblas_gemm.hip.cpp

```diff
@@ -274,8 +274,8 @@ void hipblas_dgemm_wrapper(char transa, char transb, int m, int n, int k,
     op_t2 = HIPBLAS_OP_T;
 
   if (!hip_alreadyAllocated_dgemm_handle) {
-    hipblasCreate(&handle_hip_dgemm);
-    hip_alreadyAllocated_dgemm_handle = true;
+    HICBLAS_CHECK(hipblasCreate(&handle_hip_dgemm));
+    hip_alreadyAllocated_sgemm_handle = true;
```

Comment:
should this not be `dgemm` instead of `sgemm`?

Reply:
of course! good catch... I will overwrite the merge commit. Isn't there a rule of thumb to never use two variables that only differ in one character?